.github/config.yml

```diff
@@ -29,5 +33,7 @@ firstPRMergeComment: |
  <p>People often get stuck at the same steps, so you might be able to help someone get unstuck, or help lead them to some documentation that'd help. Reach out and be encouraging and friendly! 😄 🎉 </p>
  <p>Read about <a href="https://github.com/publiclab/plots2/issues/3458">how to help support another newcomer here</a>, or find <a href="https://github.com/publiclab/plots2/labels/support">other ways to offer mutual support here</a>.</p>
  </details>
+ <br />
```

Comment:
Great job @TenTraicion!! Can you remove line 36? Thanks!!

Reply:
Removed the line. Kindly check @TildaDares and Thanks